vnext/PropertySheets/Autolink.targets

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- 
+  Copyright (c) Microsoft Corporation. All rights reserved.
+ Licensed under the MIT License.. 
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="RunAutolink" BeforeTargets="PrepareForBuild">
+    <Exec Command="$(AutolinkCommand) $(AutolinkCommandArgs)" ConsoleToMSBuild="true" WorkingDirectory="$(AutolinkCommandWorkingDir)" CustomWarningRegularExpression="Warning: " CustomErrorRegularExpression="Error: " />
```

Comment:
`ConsoleToMSBuild` set to true will make the output available into whatever the `ConsoleOutput` attribute gets set to, so should we capture it by setting `ConsoleOutput` to something and then Issuing a <Error /> element with that variable as text if we find an error?

Reply:
I just copied this from the BundleCommand in the Bundle.Common.targets. I just want to make sure that the warning/errors get parsed and displayed in VS.

Reply:
Removed, still does what I need.